content/news/2024-07-01-GCC2024/index.md

```diff
@@ -0,0 +1,127 @@
+
+One Villa tour group took the tram, and another group enjoyed a walk despite the warm weather. The Villa Tugendhat visit was particularly special due to its remarkable modernity for its time. Designed by German architect Ludwig Mies van der Rohe, the villa was built in just two years by the wealthy and influential Jewish Czech Tugendhat family. It is recognized as a UNESCO World Heritage Site.
+
+During the city tour, we explored the main sights of Brno's city center with guided tours organized by the Brno Tourist Information Centre. We visited the Old Town Hall, St. Peter's Cathedral, as well as the Freedom and Market Squares. The experience was enjoyable and provided a rich insight into Brno's history and culture.
+
+### Conference Nights Activities
+
+Beyond the enriching presentations and discussions, conference nights were filled with fun and camaraderie. Groups of attendees ventured out to the Bouldering Rock Climbing Hall, testing their climbing skills and bonding over shared challenges. Others enjoyed friendly competitions at local billiards halls.
+
+![GCC2024 Billiards](gcc2024billiards.jpg)
+
+Dinner outings were a highlight, with groups exploring Brno’s culinary scene, sampling delicious Mexican and Indian cuisine. The evenings often concluded with open-air talks, where participants relaxed and enjoyed local beers and [Kofola](https://en.wikipedia.org/wiki/Kofola) at various pubs and bars, further strengthening connections and creating memorable experiences.
+
+![GCC2024 Beer Nights](gcc2024beer.jpg)
+
+## Thank you: Scientific Committee Members
+
+* Alex Ostrovsky, Johns Hopkins University, United States
+* Beatriz Serrano-Solano, Euro-BioImaging ERIC, Germany
+* Elliott Price, Masaryk University, Czech Republic
+* Enis Afgan, Johns Hopkins University, United States
+* Hans-Rudolf Hotz, Friedrich Miescher Institute for Biomedical Research, Switzerland
+* Lucille Delisle, Swiss Federal Institute of Technology, Switzerland
+* Wendi Bacon, The Open University, England
+
+## Thank you: Organizers
+
+### Conference
+
+* Björn Grüning, University of Freiburg, Germany
+* Enis Afgan, Johns Hopkins University, United States
+* Helge Hecht, Masaryk University, Czech Republic
+* Jenn Vessio, Johns Hopkins University, United States
+* Natalie Whitaker, Johns Hopkins University, United States
+
+### Training
+
+* Tyler Collins, Johns Hopkins University, United States
+* Assunta DeSanto, Penn State University, United States
+
+### CoFest
+
+* Ahmed Awan, Johns Hopkins University, United States
+* Björn Grüning, University of Freiburg, Germany
+
+## Thank you to All GCC 2024 sponsors:
+
+* [JXTX Foundation](https://jxtxfoundation.org/)
+* [Worfklow4Metabelomics](https://workflow4metabolomics.org/)
+* [GalaxyWorks](https://galaxyworks.io/)
+* [GigaScience](https://academic.oup.com/gigascience)
+* [KWS](https://www.kws.com/corp/en/)
+* [Limagrain](https://www.limagrain.com/en)
+* [deNBI](https://www.denbi.de/)
```

Comment:
@erxleben can we here highlight a bit de.NBI and also mention that ESG was present and had a "booth" :)